alacritty_terminal/src/grid/storage.rs

```diff
@@ -192,7 +192,7 @@ impl<T> Storage<T> {
     #[inline]
     /// Compute actual index in underlying storage given the requested index.
     fn compute_index(&self, requested: usize) -> usize {
-        debug_assert!(requested < self.len);
+        debug_assert!(requested <= self.len);
```

Comment:
This would allow index out of bounds. If this is triggered, the code triggering it should be fixed.